services/director-v2/docker-compose-extra.yml

```diff
@@ -22,10 +22,10 @@ services:
         "-c",
         "log_duration=true",
         "-c",
-        "log_line_prefix=[%p] [%a] [%c] [%x] ",
+        "log_line_prefix=[%p] [%a] [%c] [%x] "
       ]
   rabbit:
-    image: itisfoundation/rabbitmq:3.8.0-management
+    image: itisfoundation/rabbitmq:3.11.2-management
```

Comment:
At some point we should also get rid of these ``docker-compose-extra.yml`` ( i am the one to blame ... )!

Reply:
;) yep.